### ISSUE TITLE
Tabbed Nav – aria-orientation should be consistent

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#2463]
Angular and React are applying it to different elements. Vanilla doesn’t appear to be working at all. Figure out what is correct and apply it to all 3 frameworks.

Bob: (as far as I can tell, aria-orientation doesn’t actually do anything)

Point: 5
